RecoHI/HiEgammaAlgos/interface/CxCalculator.h

```diff
@@ -0,0 +1,46 @@
+#ifndef CxCalculator_h
+#define CxCalculator_h
+
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/EventSetup.h"
+#include "FWCore/Framework/interface/ESHandle.h"
+
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/Utilities/interface/InputTag.h"
+
+#include "DataFormats/EgammaReco/interface/BasicClusterFwd.h"
+#include "DataFormats/EgammaReco/interface/SuperClusterFwd.h"
+
+#include "Geometry/CaloGeometry/interface/CaloGeometry.h"
+#include "Geometry/Records/interface/IdealGeometryRecord.h"
+
+
+class CxCalculator
```

Comment:
similar to comments to the dataFormats, please use more intuitive/descriptive class names.